lnpbp-0024.md

```diff
@@ -57,7 +60,7 @@ interface RGB24
                    invalidRoot |
                    incompleteRegistry
 
-data Hostname :: [RGBTypes.AlphaNumDash ^ 1..63]
+data Hostname :: [RGBContract.AlphaNumDash ^ 1..63]
```

Comment:
I think `AlphaNumDash` is a part of `Std` now:
https://github.com/strict-types/registry/blob/2e93f75bcbd4b1ef501a2b14bec1193e839648f4/UBIDECO.org/Std/Std%400.1.0.sty#L69-L84